client/src/modules/stock/exit/exit.js

```diff
@@ -412,19 +469,36 @@ function StockExitController(
     if (!vm.movement.entity.uuid && vm.movement.entity.type !== 'loss') {
       return Notify.danger('ERRORS.ER_NO_STOCK_DESTINATION');
     }
+
+    if (hasDuplicatedLots()) {
+      // notify on the concerned row
+      errorLineHighlight();
+      return Notify.danger('ERRORS.ER_DUPLICATED_LOT', 20000);
```

Comment:
Why is this such a long notification? 20 seconds?

Reply:
I would like to show the message for a long time to be sure that the user read that and he can close the message himself